curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -83,12 +83,18 @@ int DiskCacheManager::Init(S3Client *client,
     SetDiskInitUsedBytes();
     SetDiskFsUsedRatio();
 
+    // init throttle
     ReadWriteThrottleParams params;
-    params.iopsWrite = ThrottleParams(option.diskCacheOpt.maxFlushIops, 0, 0);
-    params.bpsWrite = ThrottleParams(option.diskCacheOpt.maxFlushBytes, 0, 0);
-    params.iopsRead = ThrottleParams(option.diskCacheOpt.maxReadFileIops, 0, 0);
-    params.bpsRead = ThrottleParams(option.diskCacheOpt.maxReadFileBytes, 0, 0);
-
+    params.iopsWrite = ThrottleParams(
```

Comment:
At the beginning, you need to make sure that the cache disk will be used by several mount points？
Consider using GFlag to define parameter support for online changes to support situations where the cache disk is shared by multiple FUSE.

Reply:
fix